src/services/getResponseDataService.ts

```diff
@@ -0,0 +1,5 @@
+import { instance } from '../api/api';
+
+export const fetchDataService = (url: string, options?: string) => {
+  return instance.get(options ? `${url}/${options}` : url);
+};
```

Comment:
when I was writing about the services approach I meant something like this:
/services
     articlesService.ts (here we have getArticle, getArticles etc.)
     tagsService.ts (getTags)